administrator-manual/en/base_system2.rst

```diff
@@ -0,0 +1,247 @@
+
+
+.. _services-section:
+
+Services
+--------
+
+A :index:`service` is a software running on the firewall itself.
+
+Each service can have a list of "open" ports accepting local or remote connections.
+To control which zones or host can access a network service, see :ref:`firewall-section`.
+
+Existing services can be started and stopped directly from the :guilabel:`Services` page.
+
+
+.. _storage-section:
+
+Storage
+-------
+
+The :index:`storage` section configures and monitors disks.
+The administrator can mount new local or remote disks, manage RAID arrays and LVM volumes.
+
+
+.. _ssh-section:
+
+SSH
+---
+
+The :index:`ssh` page displays the number of current SSH connection.
+From this section the administrator can change the OpenSSH listening port, disable root login or
+password authentication.
+
+.. _settings-section:
+
+Settings
+--------
+
+The :index:`settings` page allows the configuration some options which can impact multiple system applications.
+
+Smart host
+^^^^^^^^^^
+
+Many system applications, like cron, can generate mail notification.
+If the server can't directly deliver such mails, the administrator can configure
+a SMTP relay.
+When the smarthost is enabled, all mail messages will be delivered to the configured server.
+
+Email notifications
+^^^^^^^^^^^^^^^^^^^
+
+As default, notifications are sent to the local root maildir.
+The administrator can change the root forward address adding one or more mail address to the :guilabel:`Destination` field.
+
+It's also a good practice to set a custom :guilabel:`Sender address`: messages from the root user (like cron notifications)
+will be sent using the specified address.
+A good value could be: ``no-reply@<domain>`` (where ``<domain>`` is the domain of the server).
+If not set, messages will be sent using ``root@<fqdn>`` as sender address.
+
+Server Manager
+^^^^^^^^^^^^^^
+
+As default, access to the Server Manager is granted from all firewall zones.
+From this section the administrator can restrict the access to the Server Manager only to
+a list of trusted IP addresses.
+
+Log files
+^^^^^^^^^
+
+All log files are managed by :index:`logrotate`. Logrotate is designed to ease administration of a large numbers of log files.
+It allows automatic rotation, compression, and removal of log files. Each log file may be handled daily, weekly, monthly.
+
+The administrator can set logrotate defaults from this page. The configuration will apply to all applications.
+But please note that some applications can override such configuration to meet specific needs.
+
+Configuration hints
+^^^^^^^^^^^^^^^^^^^
+
+Most Server Manager pages can display some configuration hints to help guide the administrator
+on a better system configuration.
+Hints are just suggestions and can be disabled from this menu.
+
+Password change
+^^^^^^^^^^^^^^^
+
+The setting page also include the password change for local users, including root.
```

Comment:
setting -> Settings
includes
a panel to let users change their password, including the *root* user.